english/marvel-at-dota2-top-plays/marvel-at-top-plays.md

```diff
@@ -0,0 +1,6 @@
+You open your youtube favorite.
+
+Type in DC Top Plays.
+
+Start watching a video in the [playlist.](https://youtu.be/j-Ts28rdZm4?list=PL981BABEC1803C00D)
+then cry yourself to sleep because [you will never be as good.](https://youtu.be/t0zUQ8nMb9E)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.